client/cmd/dexc/main.go

```diff
@@ -175,3 +178,36 @@ func mainCore() error {
 
 	return nil
 }
+
+// promptShutdown checks if there are active orders and asks confirmation to
+// shutdown if there are. The return value indicates if it is safe to stop Core
+// or if the user has confirmed they want to shutdown with active orders.
+func promptShutdown(clientCore *core.Core) bool {
+	err := clientCore.Logout()
+	if err == nil {
+		return true
+	}
+	if !errors.Is(err, core.ActiveOrdersLogoutErr) {
+		log.Errorf("unable to logout: %v", err)
+		return false
```

Comment:
true?

Reply:
Yah, if it's not `hasActiveOrders`, just some error locking a wallet, OK to log the issue and say true (shutdown is safe).